client/src/legacy/HtmlEditorField.js

```diff
@@ -288,7 +288,7 @@ jQuery.entwine('ss', function($) {
     /**
      * Constructor: onmatch
      */
-    onadd: function() {
+    onmatch: function() {
```

Comment:
This change is throwing errors when I try and add new content blocks now (testing https://github.com/dnadesign/silverstripe-elemental/pull/372)